.travis.yml

```diff
@@ -84,7 +84,20 @@ jobs:
       virt: vm
       env:
         - CIBW_BUILD: cp310-manylinux_aarch64
-      install: python3 -m pip install cibuildwheel==2.4.0
+      install: python3 -m pip install cibuildwheel==2.9.0
+      script: |
+        cibuildwheel --output-dir wheelhouse
+        source ./tools/wheels/upload_wheels.sh
+        set_travis_vars
+        set_upload_vars
+        upload_wheels # Will be skipped if not a push/tag/scheduled build
+    - python: "3.11"
+      os: linux
+      arch: arm64
+      virt: vm
+      env:
+        - CIBW_BUILD: cp311-manylinux_aarch64
```

Comment:
The `CIBW_BUILD` directive determines which build is performed. You should be able to use whatever python you want in the `- python:` directive (line 94, 81, 68) since the build actually occurs inside a manylinux_aarch64 docker image, i.e. manylinux 2014.